src/net/p2p/encoding.cpp

```diff
@@ -178,7 +178,6 @@ namespace P2P {
     std::unordered_map<NodeID, NodeType, SafeHash> nodes;
 
     BytesArrView data = message.message();
-    Utils::logToDebug(Log::P2PRequestDecoder, __func__, std::string("Data size: ") + std::to_string(data.size()) + " data: " + Hex::fromBytes(data).get());
```

Comment:
Suppose this was meant to be converted to a `Logger::logToDebug()` call instead of just being deleted? If not please ignore.